docs/intro.rst

```diff
@@ -41,7 +41,8 @@ As a simple "Hello World!"-example we will try to learn the Pareto problem:
    gen = ChoiceDatasetGenerator(dataset_type='pareto',
                                    n_objects=30,
                                    n_features=2)
-   X_train, Y_train, X_test, Y_test = gen.get_single_train_test_split()                     
+   X_train, Y_train, X_test, Y_test = gen.get_single_train_test_split()
+
```

Comment:
This seems unrelated, was the change intentional?

Reply:
That is unrelated indeed. But we should fix it separately and get a proper docs linter up and running.